civis_jupyter_notebooks/assets/jupyter_notebook_config.py

```diff
@@ -42,9 +23,51 @@
         platform_persistence.logger.warn('Killing the notebook process b/c of a startup issue')
         os.kill(os.getpid(), signal.SIGTERM)
 
-    # Install requirements.txt
-    REQUIREMENTS_PATH = os.path.join(ROOT_DIR, 'requirements.txt')
-    if os.path.isfile(REQUIREMENTS_PATH):
-        platform_persistence.logger.info('installing requirements.txt packages')
-        pip.main(['install', '-r', REQUIREMENTS_PATH])
-        platform_persistence.logger.info('requirements.txt installed')
+
+def find_and_install_requirements(requirements_path, c):
+    try:
+        platform_persistence.find_and_install_requirements(requirements_path)
+    except NotebookManagementError as e:
+        # redirect to log file if pip fails
+        error_msg = "Unable to install requirements.txt:\n" + str(e)
+        platform_persistence.logger.info(error_msg)
+        with open(os.path.join(ROOT_DIR, 'civis-notebook-logs.log'), 'w') as f:
```

Comment:
Take a look at the `civis-git-clone` script. You can simply write to the `civis-notebook-logs.log` file using the `file_logger`.